mne/preprocessing/eyetracking/eyetracking.py

```diff
@@ -9,11 +9,68 @@
 from ...epochs import BaseEpochs
 from ...evoked import Evoked
 from ...io import BaseRaw
-from ...utils import _check_option, _validate_type, logger, warn
+from ...utils import _check_option, _validate_type, logger, verbose, warn
 from .calibration import Calibration
 from .utils import _check_calibration
 
 
+@verbose
+def find_blinks(
+    inst,
+    *,
+    chs_src=None,
+    method="by_dropout",
+    dropout_value=None,
+    description="BAD_blink",
+    chs_dest=None,
+    verbose=None,
+):
+    """Find blinks in eye-tracking data and create corresponding annotations.
+
+    Parameters
+    ----------
+    inst : instance of Raw
+        The data instance to use for finding blinks. Must contain pupil channels.
+    chs_src : list | None
+        A list of channel names that will be used to find blinks. None (default) will
+        result in selecting all channels of type ``pupil``. See Notes for more
+        information.
+    method : str
+        Which method to use to find blinks in ``inst``. Currently the only supported
+        method is ``'by_dropout'`` (default), which uses the value specified via
+        ``dropout_value`` to identify blinks.
+    dropout_value : float | None
+        Which value in the data denotes a dropout. In Eyelink data, this is ``0``,
+        whereas for other eye-tracking data this may be ``np.nan``, or something else.
+        Defaults to None, which sets the ``dropout_value`` to ``np.nan``.
+    description : str
+        Which description to use for the blink annotations. Defaults to ``'BAD_blink'``.
+    chs_dest : list | None
+        A list of channel names that will be associated with the blink annotations.
+        None (default) and passing an empty lists will associate each blink annotation
```

Comment:
I don't love the fact that to get *all* channels annotated you pass either `None` or `[]`

Reply:
I know what you mean, and I agree in principle. We had a similar discussion in mne-bids recently:

- https://github.com/mne-tools/mne-bids/pull/1307

However, this choice is based on the `ch_names` parameter in https://mne.tools/stable/generated/mne.Annotations.html

I am open to alternative suggestions.

Reply:
> To me it's a bit unclear in what cases users might want to associate other channels with the blink annotations

users may want to associate EEG channels with blinks that are detected from pupil channels, for artifact cleaning.

But as long as this is possible, I do not mind about the exact implementation.

Reply:
> users may want to associate EEG channels with blinks that are detected from pupil channels, for artifact cleaning.

Would `ch_names=[]` be sufficient? I have no experience in oculomotor artifact correction so may have the ask the stupid question of whether those algorithms require annotations with associated EEG channel names?

Reply:
I think these are details that we can probably decide later on in the process, for now I think that @scott-huberty's proposal has received a lot of 👍, so we should go with this and revisit when it is time to merge.